packages/administration/templates/form/type/fileUpload/AbstractFileUploadType.html.twig

```diff
@@ -20,7 +20,7 @@
                 </div>
             {% endblock abstract_file_upload_widget_uploaded_files %}
 
-            <div class="position-fixed top-0 end-0 m-6 p-3 w-25" style="display: none; z-index: 1090">
+            <div class="position-fixed top-0 end-0 m-6" style="display: none; z-index: 1090">
```

Comment:
isn't it looking much as a button now? 

<img width="211" height="146" alt="Snímek obrazovky 2025-11-05 v 16 40 00" src="https://github.com/user-attachments/assets/39273377-30f0-4562-a79d-aa728851be5b" />